ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+**When reporting a bug, make sure you provide all required info below. Reports without this information may be closed.**
+
+
+Output of `docpub --version`:
+
+...
+
+Contents of `docpub.config` file:
```

Comment:
This seems a little dangerous because people might not be paying attention and end up posting their username and API key.

Is it really necessary to have the `docpub.config` file in issues? As of right now, there probably isn't anything that will be relevant to issues, but I can see how config options we add later could be relevant.